### PR TITLE
160 bufr table b descriptors for atmospheric composition

### PR DESCRIPTION
**Pull request by amilan17**
#160

### CODE REVIEW

File: BUFRCREX_TableB_en_08.csv [hunk 1]
```diff
@@ -41,6 +41,7 @@ ClassNo,ClassName_en,FXY,ElementName_en,BUFR_Unit,BUFR_Scale,BUFR_ReferenceValue
 08,Significance qualifiers,008042,Extended vertical sounding significance,Flag table,0,0,18,Flag table,0,6,,,Operational
 08,Significance qualifiers,008043,Atmospheric chemical or physical constituent type,Code table,0,0,8,Code table,0,3,(see Note 7),169,Operational
 08,Significance qualifiers,008044,CAS registry number,CCITT IA5,0,0,88,Character,0,11,,,Operational
+08,Significance qualifiers,008045,Particulate matter characterization,Code table,0,0,8,Character,0,2,,,Operational
```
Comment (amilan17) [inline BUFRCREX_TableB_en_08.csv:44]: change name?

File: BUFRCREX_CodeFlag_en_08.csv
[the remark this answers is not in the document]
Comment (marijanacrepulja) [reply BUFRCREX_CodeFlag_en_08.csv:1]: @amilan17 Thank you for suggestion, I have updated branch with a new name.

File: BUFR_TableD_en_11.csv [hunk 2]
```diff
@@ -217,3 +217,21 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
 11,Single level report sequences (conventional data),311012,(BUFR template for aircraft ascent/descent profile with latitude and longitude given for each level),(Aircraft ascent/descent profile data for one level with lat. long. Indicated),201000,Cancel change data width,,,,Operational
 11,Single level report sequences (conventional data),311012,(BUFR template for aircraft ascent/descent profile with latitude and longitude given for each level),(Aircraft ascent/descent profile data for one level with lat. long. Indicated),013003,Relative humidity,"Value derived from Mixing ratio, limited to ≤ 100%",,,Operational
 11,Single level report sequences (conventional data),311012,(BUFR template for aircraft ascent/descent profile with latitude and longitude given for each level),(Aircraft ascent/descent profile data for one level with lat. long. Indicated),033026,Moisture quality,Code table ,,,Operational
+11,Single level report sequences (conventional data),311013,(Uncrewed Aircraft Systems),,301150,WIGOS identifier,,,,Operational
```
Comment (amilan17) [inline BUFR_TableD_en_11.csv:220]: @marijanacrepulja This change belongs to issue #163, which I already merged into FT2023-1. Should I delete this change?

Comment (marijanacrepulja) [reply BUFR_TableD_en_11.csv:220]: @amilan17 If is already in the branch for https://github.com/wmo-im/BUFR4/issues/163 then it is sufficient.

Comment (amilan17) [reply BUFR_TableD_en_11.csv:220]: thx

**Comment by amilan17** [comment]
#160 merged into FT2023-2

**Pull request merged**